packages/lexical-editor/src/nodes/webinyNodes.ts

```diff
@@ -11,6 +11,8 @@ import { TypographyElementNode } from "~/nodes/TypographyElementNode";
 import { WebinyListNode } from "~/nodes/list-node/WebinyListNode";
 import { WebinyListItemNode } from "~/nodes/list-node/WebinyListItemNode";
 import { WebinyQuoteNode } from "~/nodes/WebinyQuoteNode";
+import { BaseParagraphNode } from "~/nodes/BaseParagraphNode";
+import { ParagraphNode } from "lexical";
```

Comment:
We've been talking about this on our call.. But, you know what, let's do this like the following:

```ts
import { ParagraphNode } from "~/nodes/ParagraphNode";
import { ParagraphNode as LexicalParagraphNode } from "lexical";
```

Let's also apply the same approach in `~/nodes/ParagraphNode`.

Hope I did not miss anything.